src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -3594,4 +3644,191 @@ make_NetworkOPs (NetworkOPs::clock_type& clock, bool standalone,
         job_queue, ledgerMaster, parent, journal);
 }
 
+
+//------------------------------------------------------------------------------
+
+struct AccountTxPager_test : TestSuite
+{
+    void run() override
+    {
+        const char* path = std::getenv("TEST_FIXTURES");
```

Comment:
Would be nice if we had some way of getting fixture path, etc, via a TestSuite method?